app/pages/sections/routes.js

```diff
@@ -74,5 +74,20 @@ export const sectionRoutes = (authProvider, addContext, sessionManager) => {
     res.render('pages/sections/commencement-date/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.post('/commencement-date', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
+    const { orderId } = req.params;
+
+    const response = await putCommencementDate({
+      orgId: req.user.primaryOrganisationId,
+      orderId,
+      data: req.body,
+      accessToken: extractAccessToken({ req, tokenType: 'access' }),
+    });
+
+    if (!response.success) logger.info(`validation error: ${JSON.stringify(response)}`);
```

Comment:
take it you are going to change this later on to follow the pattern of handling the happy path?

Reply:
yep